cmd/zetacored/root.go

```diff
@@ -114,6 +114,8 @@ func initTmConfig() *tmcfg.Config {
 		cfg.Mempool.Version = tmcfg.MempoolV1
 	}
 
+	cfg.DBBackend = "pebbledb"
```

Comment:
I think we moved to rocksDB at some point to improve performace ? 
And compared pebbleDB and rocksDB when doing that .  Has something changed since then ?

Reply:
All our nodes run on pebbledb, so if you want to restore from our snapshots (the only thing that's really supported) then you also have to use pebbledb.